table_engine/src/table.rs

```diff
@@ -487,6 +487,11 @@ pub trait Table: std::fmt::Debug {
     /// Get table's statistics.
     fn stats(&self) -> TableStats;
 
+    /// Whether the columns used in filter expr can be pushdown
+    fn support_pushdown(&self, _read_schema: &Schema, _columns: &[String]) -> bool {
```

Comment:
`read_schema` seems useless here, and the column name should be converted if there is any column alias.

And shall we use `Iterator<&str>` as the type for `columns`  to make it more general?

Reply:
Add comments explaining why.